pkg/logger/zap.go

```diff
@@ -15,12 +15,15 @@ import (
 // The package logger can only be init once, so subsequent calls to this method
 // won't have any effect.
 func InitZap(level int, opts ...LoggerOpt) error {
-	cfg := zap.NewDevelopmentConfig()
-	cfg.Level = zap.NewAtomicLevelAt(zapcore.Level(-1 * level))
+	filename := fmt.Sprintf("./eksa-cli-%s.log", time.Now().Format("2006-01-02T15:04:05"))
+	cfg := Config{
+		Level:         NewAtomicLevelAt(level),
+		EncoderConfig: zap.NewDevelopmentEncoderConfig(),
+		OutputPaths:   []string{filename},
+	}
+
 	cfg.EncoderConfig.EncodeLevel = nil
 	cfg.EncoderConfig.EncodeTime = NullTimeEncoder
-	cfg.DisableCaller = true
-	cfg.DisableStacktrace = true
```

Comment:
are we replacing these two with something else?

Reply:
no, there is not a need to I think. Under the hood of zap's Config in the buildOptions method [here](https://github.com/uber-go/zap/blob/master/config.go#L198), if this was true, it would skip a step. Since, we want to skip it anyway, i didn't include it in our custom config.

Reply:
Does that mean that the new way of building a config will result in the same behavior as before?

Reply:
Yes, that was the goal

Reply:
Yes, it should